src/pages/Dashboard/Apps/Add/Card/index.jsx

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { observer } from 'mobx-react';
+import classNames from 'classnames';
+import { translate } from 'react-i18next';
+
+import { Icon, DocLink } from 'components/Base';
+
+import styles from './index.scss';
+
+@translate()
+@observer
+export default class CreateAppCard extends Component {
+  static propTypes = {
+    appCreateStore: PropTypes.object,
```

Comment:
I think this `appCreateStore` is no need. External store will be passed down to this component.

Reply:
ok